lib/active_storage_validations/matchers/content_type_validator_matcher.rb

```diff
@@ -57,7 +57,7 @@ def allowed_types
       end
 
       def rejected_types
-        @rejected_types || (Mime::LOOKUP.keys - allowed_types)
+        @rejected_types || (Marcel::TYPES.keys - allowed_types)
```

Comment:
```ruby
Mime::LOOKUP.keys.count
# => 45
```
```ruby
Marcel::TYPES.keys.count
# => 892
```
It influences the speed of tests

Reply:
how big is the impact?

Reply:
Doesn't look like a big impact for
```sh
$ BUNDLE_GEMFILE=gemfiles/rails_6_1.gemfile bundle exec rake test
```
before:
```
Finished in 5.077188s, 12.8024 runs/s, 23.0443 assertions/s.
65 runs, 117 assertions, 0 failures, 0 errors, 0 skips
```
after
```
Finished in 5.293181s, 13.0356 runs/s, 22.6707 assertions/s.
69 runs, 120 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
https://github.com/igorkasyanchuk/active_storage_validations/issues/127